concepts/Common/TypeScript Guides/15 Events/00 Events.md

```diff
@@ -0,0 +1 @@
+TypeScript offers tools to handle events with precision and type safety. This section explains how to read events in DevExtreme components.
```

Comment:
Not sure what you mean by "precision". Should we explain that? Or maybe remove that part?